nextest-runner/src/config/overrides.rs

```diff
@@ -1029,10 +1029,10 @@ mod tests {
     }
 
     #[test_case(
-        indoc! {r#"
+        indoc! {r"
             [[profile.default.overrides]]
             retries = 2
-        "#},
+        "},
```

Comment:
Thank you for the PR!

I would rather not drop the `#` from instances containing configuration snippets, since they can easily have double-quotes in the future. In general I don't think an extra `#` on both sides is a particularly major concern, but if you'd like to update this PR with a version that doesn't alter configuration snippets I would be happy to land that.

Reply:
You're welcome! Which ones are the configuration snippets?

I'll push a trimmed-down version of the original commit that should hopefully avoid them.

I can take another pass on these changes after that if any more changes are necessary and it doesn't take up too much of your time.